<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="TR-JDGMXY2" failures="1" tests="1" name="SignoffTest" time="0.034" errors="0" timestamp="2021-02-23T16:08:43 ART">
  <testcase classname="com.trgr.quality.maf.tests.SignoffTest" name="displayOfSignOffSummary" time="0.003">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Exception in Test expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Exception in Test expected [true] but found [false]
at com.trgr.quality.maf.tests.SignoffTest.displayOfSignOffSummary(SignoffTest.java:150)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- displayOfSignOffSummary -->
  <testcase classname="com.trgr.quality.maf.tests.SignoffTest" name="NavigationfromSignofftologinPage" time="0.0">
    <ignored/>
  </testcase> <!-- NavigationfromSignofftologinPage -->
  <testcase classname="com.trgr.quality.maf.tests.SignoffTest" name="Signoff" time="0.0">
    <ignored/>
  </testcase> <!-- Signoff -->
</testsuite> <!-- SignoffTest -->
